data/src/main/java/com/ronnie/data/models/ImagesEntity.kt

```diff
@@ -15,5 +15,5 @@ data class ImagesEntity(
     val user: String,
     val user_id: Int,
     val views: Int,
-    var searchTerm: String? = null
+    var searchTerm: String? = "cat"
```

Comment:
Why cat?

Reply:
Was using that for testing,I can just set that to null and it wil still work